### PR TITLE
9.0 enterprise settings adr

### PR DESCRIPTION
**Pull request by antonylesuisse**

### CODE REVIEW

File: addons/project/res_config.py [hunk 1]
```diff
@@ -54,6 +54,18 @@ class project_configuration(osv.osv_memory):
...
             help="Odoo will generate an email alias at the project creation from project name."),
+        'module_project_timesheet_synchro': fields.selection([
+            (0, "Do not install the Chrome app for timesheets"),
+            (1, "Install the Chrome app for timesheets"),
+            ], "Timesheets Apps"),
+        'module_project_forecast': fields.selection([
+            (0, "Disable forecasts"),
+            (1, "Enable forecasts"),
+            ], "Forecasts"),
+        'module_web_gantt': fields.selection([
```
Comment (antonylesuisse) [inline addons/project/res_config.py]: Remove

File: addons/website/models/res_config.py [hunk 2]
```diff
@@ -24,6 +24,18 @@ class website_config_settings(osv.osv_memory):
         'cdn_activated': fields.related('website_id', 'cdn_activated', type="boolean", string='Use CDN'),
         'cdn_url': fields.related('website_id', 'cdn_url', type="char", string='CDN Base URL'),
         'cdn_filters': fields.related('website_id', 'cdn_filters', type="text", string='CDN Filters'),
+        'module_website_form_editor': fields.selection([
+            (0, "Disable the form builder"),
+            (1, "Enable the form builder"),
+            ], "Form builder"),
+        'cta_blocks': fields.selection([
```
Comment (antonylesuisse) [inline addons/website/models/res_config.py]: Remove

File: addons/web/static/src/js/views/form_widgets.js [hunk 3]
```diff
@@ -1629,6 +1629,73 @@ var FieldToggleBoolean = common.AbstractField.extend({
...
+
+            dialog.on('closed', this, function() {
+                this.$checkbox.prop("checked", false);
+            });
+        }
+    },
+
+    render_value: function() {
+        this._super();
+        this.$el.append(" <span class='label label-primary'>Enterprise</span>");
```
Comment (antonylesuisse) [inline addons/web/static/src/js/views/form_widgets.js]: Remvoe

**Comment by mart-e** [comment]
merged at ed1bbbb8d04a67048f46f015484cc3ec2508818f

**Pull request closed without merging**